File: Programming Basics with Python/04 PB Python for loop/More exercises/06_Bills.py
mounts = int(input())
water = 20 * mounts
internet = 15 * mounts
total_electricity = 0
total_other_costs = 0
for cost in range(mounts):
    electricity = float(input())
    total_electricity += electricity
    other_costs = (20 + 15 + electricity) * 1.2
    total_other_costs += other_costs
average = (water + internet + total_electricity + total_other_costs) / mounts
print(f'Electricity: {total_electricity:.2f} lv')
print(f'Water: {water:.2f} lv')
print(f'Internet: {internet:.2f} lv')
print(f'Other: {total_other_costs:.2f} lv')
print(f'Average: {average:.2f} lv')

File: JS Basics/04 For-Loop - More Exercises/06. Bills.js
function bills(input) {
    let months = Number(input[0]);
    let water = 20 * months;
    let internet = 15 * months;
    let other = 0
    let electricity = 0
    for (let i = 1; i <= months; i++) {
        let electricityPerMonth = Number(input[i]);
        other += (20 + 15 + electricityPerMonth) * 1.2;
        electricity += electricityPerMonth;
    }
    let avrPerMonth = (water + internet + electricity + other) / months;
    console.log(`Electricity: ${electricity.toFixed(2)} lv`);
    console.log(`Water: ${water.toFixed(2)} lv`);
    console.log(`Internet: ${internet.toFixed(2)} lv`);
    console.log(`Other: ${other.toFixed(2)} lv`);
    console.log(`Average: ${avrPerMonth.toFixed(2)} lv`);
}
bills([8, 123.54, 231.54, 140.23, 100, 122.4, 430, 178.52, 64.2])